internal/interactivity/hiddenCommands.go

```diff
@@ -0,0 +1,10 @@
+package interactivity
+
+var HiddenCommands = []string{
+	"auth",
```

Comment:
Are you intentionally keeping help there. Just checking

Reply:
help is already ignored

Reply:
oh
my bad

Reply:
adding that now